Animation.cs

```diff
@@ -38,9 +50,18 @@ public Animation(List<AnimationFrame> frames)
         }
     }
 
+    /// <summary>
+    /// Loads animation from ImageSharp format.
+    /// </summary>
+    /// <param name="image">The actual animation parsed by ImageSharp.</param>
+    /// <param name="format">The animation format details.</param>
+    /// <exception cref="NotSupportedException"></exception>
```

Comment:
No need for mentioning this exception.

Reply:
Done by 9836d106c39bcaab95f2aa372853a5d78ef2e988.